dataflowengineoss/build.sbt

```
@@ -1,6 +1,6 @@
 name := "dataflowengineoss"
 
-dependsOn(Projects.semanticcpg, Projects.x2cpg)
+dependsOn(Projects.semanticcpg, Projects.x2cpg % "compile->compile;test->test")
```

```suggestion
dependsOn(Projects.semanticcpg, Projects.x2cpg, Projects.x2cpg % "test->test")
```

The `;` separator led to problems with intellij in the past - splitting them up was the fix back then. Not sure if the problem still exists, but better play it safe. 

Just noticed that this occurs in other places again - I'll send a PR for that after this is merged.